test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -444,11 +444,11 @@ describe("BENEFICIAL OWNER GOV controller", () => {
       expect(mockSaveAndContinue).not.toHaveBeenCalled();
     });
 
-    test(`renders the current page ${config.BENEFICIAL_OWNER_GOV_PAGE} with only INVALID_DATE error when invalid characters are used`, async () => {
+    test(`renders the current page ${config.BENEFICIAL_OWNER_GOV_PAGE} with only INVALID_DATE error when invalid date is used`, async () => {
       const beneficialOwnerGov = { ...REQ_BODY_BENEFICIAL_OWNER_GOV_FOR_DATE_VALIDATION };
-      beneficialOwnerGov["start_date-day"] =  "a";
-      beneficialOwnerGov["start_date-month"] = "b";
-      beneficialOwnerGov["start_date-year"] = "c";
+      beneficialOwnerGov["start_date-day"] =  "11";
```

Comment:
This looks like a valid test but testing when non-digits are supplied also looks valid. Should/could we keep the original test too?

Reply:
i can add an extra test to cover the previous scenario of non-digits

Reply:
updated